config/teleop.yaml

```diff
@@ -21,6 +21,8 @@ teleop:
     forward_back:
       multiplier: 1
     left_right:
+      # Can be disabled here since left_right axis sucks
+      enabled: 1
```

Comment:
hey im going to change it so it's disabled by default

Reply:
Yea that makes sense